src/components/App/App.tsx

```diff
@@ -1,8 +1,31 @@
-import React from "react";
+import React, { FC } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+// import { useDispatch } from "react-redux";
```

Comment:
Better to avoid pushing commented code.
If it's needed I suggest to add 
// TODO ...

If it's nit needed you can remove this part from tutorial